### ISSUE TITLE
[METEOR 1.5.1] Meteor doesn't detect changes

### ISSUE THREAD

**Issue by dnish** [meteor/meteor#9005]
Since the last update Meteor stops detecting changes when I change a file during the rebuilding. I still get the old version and need to add an empty space to force a new rebuild. This worked fine before and Meteor automatically started a new rebuild when I've changed a file during a rebuild.

METEOR 1.5.1
macOS Sierra (10.12.6)  

Packages:
```
meteor-base@1.1.0             # Packages every Meteor app needs to have
mobile-experience@1.0.4       # Packages for a great mobile UX
mongo@1.1.19                   # The database Meteor supports right now
reactive-var@1.0.11            # Reactive variable for tracker
tracker@1.1.3                 # Meteor's client-side reactive programming library
es5-shim@4.6.15                # ECMAScript 5 compatibility for older browsers
ecmascript@0.8.1              # Enable ECMAScript2015+ syntax in app code
shell-server@0.2.4            # Server-side component of the `meteor shell` command

accounts-password@1.4.0
ssrwpo:uglifyjs2
juliancwirko:postcss
react-meteor-data
stevezhu:lodash
fourseven:scss
static-html
jquery
ground:db
```

**Comment by benjamn** [comment]
With any luck this will be fixed by #9007.

**Comment by dnish** [comment]
Ah great, I've also noticed that I get building errors more frequently (f.e. Meteor is not defined in client console with a lot of other errors). Then I need to force another rebuild to make it work again.

**Comment by benjamn** [comment]
Can you test this after running `meteor update --release 1.5.2-beta.10` in your app?

**Comment by theodorDiaconu** [comment]
Had this issue also.

**Comment by dnish** [comment]
@benjamn Seems to work better now after upgrading to beta 10.

**Comment by abernix** [comment]
@dnish Glad to hear it!  I'll close this since the fix will be released with Meteor 1.5.2.  Anyone interested in helping to test that series of Meteor, please follow #8913 for updates (plus the latest betas and release candidates)
